lib/kontena/plugin/cloud/platform/wizard_command.rb

```diff
@@ -22,12 +24,19 @@ def execute
 
     self.name = prompt.ask("Name:") unless self.name
     self.organization = prompt_organization unless self.organization
-    self.region = prompt_region unless self.region
-    self.initial_size = prompt_initial_size unless self.initial_size
+    self.type = prompt_type unless self.type
```

Comment:
could just be `def default_type    prompt.select ....  end`

Reply:
I know, but this way I have more control on things.